.changeset/thirty-shrimps-mix.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': patch
+---
+
+`TransparentUpgradeableProxy`: fix transparency in case of selector clash with non-decodable calldata
```

Comment:
```suggestion
`TransparentUpgradeableProxy`: Fix transparency in case of selector clash with non-decodable calldata.
```

I'd stick to the PR title guidelines here and use `Fix` rather than `fixed`